sdk/webpubsub/azure-messaging-webpubsub/src/test/java/com/azure/messaging/webpubsub/TokenGenerationTest.java

```diff
@@ -33,16 +32,32 @@
  * Unit tests for {@link WebPubSubServiceAsyncClient#getClientAccessToken(GetClientAccessTokenOptions)
  * getAuthenticationToken} method.
  */
-public class TokenGenerationTest  extends TestBase {
+public class TokenGenerationTest extends TestProxyTestBase {
+
+    private WebPubSubServiceClientBuilder builder;
+
+    @Override
+    protected void beforeTest() {
+        builder = new WebPubSubServiceClientBuilder()
+            .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+            .retryOptions(TestUtils.getRetryOptions())
+            .hub(TestUtils.HUB_NAME);
+    }
+
+    /**
+     * Testing token generation with connection string.
+     */
     @ParameterizedTest
     @MethodSource("getTokenOptions")
+    @DoNotRecord
```

Comment:
did we ad @DonotRecord here?

Reply:
Yeah, The recording is always empty because no network calls are made when using a connection string. Seemed wasteful to have just empty ones being uploaded.

Reply:
If this class isn't recording anything we should make this a normal unit tests and remove the `extends TestProxyTestBase`.

Reply:
The other tests in this class do make network calls. :/